rb-inotify.gemspec

```diff
@@ -1,5 +1,7 @@
 # -*- encoding: utf-8 -*-
-require_relative 'lib/rb-inotify/version'
```

Comment:
`require_relative` is not valid for Ruby 1.8 or Ruby 1.9 (Though I forget it). I got the error on this package's Travis 1.8 or 1.9 test.
The way I changed is typical way to implement this.
Ref:
https://github.com/rspec/rspec-core/blob/master/rspec-core.gemspec
https://github.com/rack-test/rack-test/blob/master/rack-test.gemspec